doc/source/ray-overview/getting-started.md

```diff
@@ -333,7 +297,7 @@ With Tune, you can launch a multi-node distributed hyperparameter sweep in less
 Tune supports any deep learning framework, including PyTorch, TensorFlow, and Keras.
 
 ````{note}
-To run this example, you will need to install the following:
+To run this example, install Ray Tune:
 
 ```bash
 pip install "ray[tune]"
```

Comment:
nit: For consistency we should have them all use `pip install -U ...` (key part of this being the `-U`)

Reply:
ack